source/reference/operator/aggregation/setWindowFields.txt

```diff
@@ -242,6 +242,12 @@ Restrictions
 
 Restrictions for the :pipeline:`$setWindowFields` stage:
 
+- Starting in MongoDB 5.1 (and 5.0.4), the :pipeline:`$setWindowFields` 
```

Comment:
Hey @ianf-mongodb , when backporting, we should only mention the versions that apply to the version the reader is on and previous.

```suggestion
- Starting in MongoDB 5.0.4, the :pipeline:`$setWindowFields` 
```